src/app/static/webpack.config.js

```diff
@@ -69,34 +78,20 @@ const commonConfig = {
     devtool: '#eval-source-map',
     plugins: [
         // make sure to include the plugin for the magic
-        new VueLoaderPlugin()
+        new VueLoaderPlugin(),
+        new HtmlWebpackPlugin({
+            hash: true,
+            template: 'public/index.ftl',
+            filename: 'index.ftl'
+        })
     ]
 };
 
-const appConfig = {...commonConfig,
```

Comment:
Also just noticed that this file could be simplified, I think must have originated from a copy paste from another webpack config with multiple configs exported.